lib/platforms/node.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var Promise = require('bluebird');
+var Mustache = require('mustache');
+var react = require('react');
+var _ = require('lodash');
+
+var TEMPLATE = "<div id='fluxapp-container' class='app' " +
```

Comment:
Is mustache necessary for a single template render? Lodash has a minimal one that is pretty straight forward and we already have the package.